front/components/items/Appointment/index.tsx

```diff
@@ -1,35 +1,52 @@
 import WrapperWidth from 'components/wrappers/WrapperWidth';
 import { isSameDay } from 'date-fns';
-import { strapiAppointmentGet } from 'lib/strapi/appointments/get';
+import { strapiAppointmentGet, strapiAppointmentGetPopulation } from 'lib/strapi/appointments/get';
 import { strapiAppointmentQuery } from 'lib/strapi/appointments/queryType';
 import { strapiTherapistsGet } from 'lib/strapi/therapists/get';
 import { strapiTherapistsQuery } from 'lib/strapi/therapists/queryType';
-import React, { useEffect, useState } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { DayPicker } from 'react-day-picker';
 import { Grid } from 'react-loader-spinner';
-import { filters, getDateOfAppointments, getTimeOfAppointments } from './helper';
 import Button from '../Button';
+import { useRouter } from 'next/router';
+import { filters } from './models';
+import {
+    formingDate,
+    getDateOfAppointments,
+    getIdOfAppointment,
+    getTimeOfAppointments,
+} from './helper';
+import { AppointmentDataContext } from 'components/wrappers/AppointmentDataContext';
 
 const Appointment = () => {
     const filters: filters = {};
+    const router = useRouter();
+    const { setAppointmentID } = useContext(AppointmentDataContext);
     const [therapists, setTherapists] = useState<strapiTherapistsQuery[]>();
     const [appointments, setAppointments] = useState<strapiAppointmentQuery[]>();
+    const [chosenTime, setChosenTime] = useState<string>();
+    const [fullDataAppointment, setFullDataAppointment] = useState<strapiAppointmentQuery[]>();
     const [chosenTherapist, setChosenTherapist] = useState<number>();
     const [chosenDate, setChosenDate] = useState<Date>();
     const [isLoading, setIsLoading] = useState(true);
+    filters.is_booked = {
```

Comment:
why won't you merge it with filters declaration by writing?

`const filters: filters = {is_booked: {$eq: false}}`